services/directions.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+var v = require('./service-helpers/validator');
+var createServiceFactory = require('./service-helpers/create-service-factory');
+
+/**
+ * Directions API service.
+ */
+var Directions = {};
+
+/**
+ * Get directions.
+ *
+ * See the [Mapbox Directions API](https://www.mapbox.com/api-documentation/#directions).
+ *
+ * @param {Object} config
+ * @param {'driving-traffic'|'driving'|'walking'|'cycling'} [config.profile=driving]
```

Comment:
I think default values that are strings in JSDoc should have quotations around them? `[config.profile="driving"]`

Reply:
I this both of them are fine [jsdoc](http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values).

For consistency I will go ahead and quotes around it.